dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganisationResource.java

```diff
@@ -209,15 +243,26 @@ private Organisation getOrganisationByIdOptionalAuth(Optional<User> user, Long i
     @Path("/all")
     @RolesAllowed({ "curator", "admin" })
     @ApiOperation(value = "List all organisations.", authorizations = { @Authorization(value = JWT_SECURITY_DEFINITION_NAME) }, notes = "Admin/curator only", responseContainer = "List", response = Organisation.class)
-    public List<Organisation> getAllOrganisations(@ApiParam(value = "Filter to apply to organisations.", required = true, allowableValues = "all, unapproved, approved") @QueryParam("type") String type) {
+    public List<Organisation> getAllOrganisations(@ApiParam(value = "Filter to apply to organisations.", required = true, allowableValues = "all, pending, rejected, approved") @QueryParam("type") String type) {
```

Comment:
Instead of `@QueryParam("type") String type`, could you make this `@QueryParam("type") ApplicationState type`? I don't see this anywhere else in the code (just String and long), so maybe there's a reason not to do it, but it would be nicer if it worked.

Reply:
Good find, I'll see if ApplicationState works.

Reply:
The issue is that the parameter has the potential to be 'all', which is not an application state.

Reply:
Does it make sense to filter by 'all'?
I didn't notice this earlier, but I would have assumed that omitting the type parameter would return return everything (subject to potential pagination rules down the road if this takes off)